src/app/open-api-generator/parser/open-api-parser.ts

```diff
@@ -0,0 +1,80 @@
+/**
+ * responsible for parsing the OpenAPI Document/Swagger Document
+ * and replacing all occurrences of ${multi-line-comment-end} substring
+ * with `*'/`
+ */
+
+import * as logger from "../../../util/logger";
+
+type TypeSchema = {
+  type?: string;
+  description?: string;
+  properties?: Record<string, TypeSchema>;
+  example?: string;
+};
+
+export function fixTypesDescriptions(
+  type: TypeSchema,
+  schemaType: string,
+  name?: string,
+) {
+  if (!type) {
+    return;
+  }
+
+  const visited = new Set<string>();
+  traverseTypes(name ? name : `${schemaType}-${name}`, type, visited);
+
+  // const typedTypes = types as Record<string, TypeSchema>;
+  // for (const currentType in typedTypes) {
+  //   const visited = new Set<string>();
+  //   traverseTypes(currentType, typedTypes[currentType]!, visited);
+  // }
+}
+
+function traverseTypes(name: string, type: TypeSchema, visited: Set<string>) {
+  if (visited.has(name)) {
+    return;
+  }
+  visited.add(name);
+
+  if (type.description && type.description.indexOf("*/") > -1) {
+    logger.warn(
+      `Encountered '*/' in description, replacing all occurrences with "*''/"`,
+    );
+    type.description = type.description.replaceAll("*/", `*''/`);
+  }
+
+  // console.log('type.example: ', type.example);
+  // console.log('typeof of example: ', typeof type.example);
+  if (
+    type.example &&
+    typeof type.example === "string" &&
+    type.example.indexOf("*/") > -1
+  ) {
+    logger.warn(
+      `Encountered '*/' in example, replacing all occurrences with "*''/"`,
+    );
+    type.description = type.example.replaceAll("*/", `*''/`);
```

Comment:
Shouldn't this be `type.example` instead of `type.description`?

Reply:
This code was not being used, hence why the tests are working even when this is incorrect. Removed this in [551aced](https://github.com/hasura/ndc-open-api-lambda/pull/21/commits/551aced67e725f9131f1872e8927e79ccd2a290b)